### PR TITLE
Portuguese comm call

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/commcalls/2024-11-06/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+attendees: Todos são bem-vindos
+slug: translation-portuguese
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: '888160235'
+resources:
+  - url:
```
Comment (yabellini) [inline content/commcalls/2024-11-06/index.md:24]:
```suggestion
  - url:  https://github.com/orgs/ropensci/projects/7/views/1
```

File: content/commcalls/2024-11-06/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+slug: translation-portuguese
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: '888160235'
+resources:
+  - url: 
+    name:
```
Comment (yabellini) [inline content/commcalls/2024-11-06/index.md:25]:
```suggestion
    name:  Tradução Português- Guia do programador
```

File: content/commcalls/2024-11-06/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+notes: 
+outputs:
+- HTML
+- Calendar 
+# vimeo:
+# - id: '888160235'
+resources:
+  - url: 
+    name:  
+    details:
```
Comment (yabellini) [inline content/commcalls/2024-11-06/index.md:26]:
```suggestion
    details: Projeto de tradução do guia de desenvolvimento de software rOpenSci para português.
```

**Review by yabellini — COMMENTED** [review]
Adding resources

**Comment by yabellini** [comment]
@beatrizmilz @ fblpalmeira @pedropark99 can you review the webpage for the comm call?  

You can check the preview here: https://deploy-preview-854--ropensci.netlify.app

I would like to publish this week to start advertising! 😄 Thank you!!

**Comment by pedropark99** [comment]
Specific page of the community call in the preview: https://deploy-preview-854--ropensci.netlify.app/commcalls/translation-portuguese/

**Review by pedropark99 — APPROVED** [review]
Looks good to me

**Comment by fblpalmeira** [comment]
A new user left a comment. This user must be approved by a Netlify team owner before comments can be displayed.

[Approve this user](https://app.netlify.com/teams/ropensci/reviewers)

**Comment by beatrizmilz** [comment]
> @beatrizmilz @fblpalmeira @pedropark99 can you review the webpage for the comm call?
> 
> You can check the preview here: https://deploy-preview-854--ropensci.netlify.app
> 
> I would like to publish this week to start advertising! 😄 Thank you!!

Sorry Yani!!

I think I missed this email.

It looks great to me!

**Pull request merged**